### PR TITLE
Add severity to KubeVirt alerts

### PR DESCRIPTION
**Pull request by sradco**
Signed-off-by: Shirly Radco <sradco@redhat.com>

**What this PR does / why we need it**:
Adding severity label to each KubeVirt alert rule.

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #

**Special notes for your reviewer**:

**Release note**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/virt-operator/resource/generate/components/crds.go [hunk 1]
```diff
@@ -792,6 +837,9 @@ func NewPrometheusRuleSpec(ns string, workloadUpdatesEnabled bool) *promv1.Prome
 						Annotations: map[string]string{
 							"summary": "Some virt-handlers failed to roll out",
 						},
+						Labels: map[string]string{
+							"severity": "critical",
```
Comment (sradco) [inline pkg/virt-operator/resource/generate/components/crds.go:841]: @ashleyschuett  Hi, Is this the correct severity for this alert?

Comment (sradco) [reply pkg/virt-operator/resource/generate/components/crds.go:841]: @EdDev @ashleyschuett ?

Comment (ashleyschuett) [reply pkg/virt-operator/resource/generate/components/crds.go:841]: I'm not sure, I think it is, but i don't know what are threshold for critical is. Do we have any documentation or guide lines about what makes something critical or what the threshold is?

**Comment by sradco** [comment]
> do we have any guidelines on what the different severity settings mean? Also, are critical and warning our only options here?
@davidvossel 
I'll add it to the monitoring guidelines.
Critical alerts is when the service is down and you loss critical functionality.
Warning alerts are for all other alerts.

File: pkg/virt-operator/resource/generate/components/crds.go (hunk 1 above)
Comment (davidvossel) [reply pkg/virt-operator/resource/generate/components/crds.go:841]: @sradco 

I think we'd want to make this a `warning` and for it to be critical if zero handlers are present.

**Review by davidvossel — APPROVED** [review]
/lgtm
/approve

@sradco as a follow up, do we need a new alert to track when no virt-handlers are online similar to what we have for virt-controller and virt-api?

**Pull request merged**